src/c++/vdo/base/slab-depot.c

```diff
@@ -2262,23 +2269,25 @@ static void load_reference_block_endio(struct bio *bio)
 }
 
 /**
- * load_reference_block() - After a block waiter has gotten a VIO from the VIO pool, load the
- *                          block.
- * @waiter: The waiter of the block to load.
+ * load_reference_block_set() - After a block waiter has gotten a VIO from the VIO pool, load
+ *                              the run of blocks.
+ * @waiter: The waiter of the first block to load.
  * @context: The VIO returned by the pool.
  */
-static void load_reference_block(struct vdo_waiter *waiter, void *context)
+static void load_reference_block_run(struct vdo_waiter *waiter, void *context)
```

Comment:
First off, the doc name ("_set") doesn't match the signature name ("_run"), so you'll need to fix that. But also, I don't really like either name. ("run" can be a noun or a verb, and I read it first as a verb, "set" has the same problem except I parse it as a noun first, so slightly better.) How about "_group"?  "_segment"? "_section"?

EDIT: We could also just leave the name as it is; it's slightly inaccurate but in context it's fairly clear what we're doing.